users/utils.py

```diff
@@ -0,0 +1,26 @@
+import jwt
+
+from django.http            import JsonResponse
+from django.core.exceptions import ObjectDoesNotExist
+
+from users.models           import User
+from my_settings            import SECRET_KEY, ALGORITHM
+
+def LogInDecorator(func):
```

Comment:
```suggestion
def login_decorator(func):
```
함수명은 스네이크 케이스로 작성하셔야 합니다.
기존에 작성했던 방법은 class 이름 지으실 때 사용하는 방법입니다.

Reply:
넵 반영하였습니다!